src/internal/base32.ts

```diff
@@ -0,0 +1,20 @@
+export const reducedBase32 = '0123456789ABCDEFGHJKMNPQRSTVWXYZ'; // Crockford's Base32 - Excludes I, L, O, and U which may be confused with numbers
```

Comment:
This doesn't need to be exported AFAICT.

```suggestion
// Crockford's Base32 - Excludes I, L, O, and U which may be confused with numbers
const reducedBase32 = '0123456789ABCDEFGHJKMNPQRSTVWXYZ';
```

Reply:
The ULID is composed of 10 characters that represents encoded timestamp, plus 16 random characters from the `reducedBase32`.
It means that the ULID method on `string` module need access to this `reducedBase32` set of characters.

```
{
  const { refDate = this.faker.defaultRefDate() } = options;
  const date = toDate(refDate);

  return encodeDate(date) + this.fromCharacters(reducedBase32, 16);
}
  ```

Reply:
Totally missed that.
Can you please convert the inline comment to a jsdoc comment?

Reply:
Question: Should we name this `crockfordsBase32Characters`?